apps/server/src/modules/user-import/uc/user-import.uc.ts

```diff
@@ -167,10 +167,11 @@ export class UserImportUc {
 		// TODO Change ImportUserRepo to DO to fix this workaround
 		const [importUsers, total] = await this.importUserRepo.findImportUsers(currentUser.school, filters, options);
 		if (total > 0) {
-			importUsers.map(async (importUser) => {
+			for (const importUser of importUsers) {
+				// TODO: Find a better solution for this loop
```

Comment:
```suggestion
				// TODO: Find a better solution for this loop
				// this needs to be synchronous, because otherwise it was leading to 
				// server crush when working with larger number of users (e.g. 1000)
```

Reply:
*crash